### ISSUE TITLE
SIDER not working at all

### ISSUE THREAD

**Issue by noagenda33** [Ylianst/MeshCentral#3278]
SIDER seems to have completely stopped working. Even if I load the smallest ISO. 4MB and still hangs for a few minutes, then disconnects. Here's screen video. Also, What is the Floppy Image for? I have only ever seen .ISO images.
https://www.screencast.com/t/Kt0HpyBoQ

**Comment by Ylianst** [comment]
Indeed, IDER is not sending or receiving any data. You can see at the top of the video, `0 in and 18 out`. Clearly not working as expected. The size of the .iso should not matter as it's sent to Intel AMT sector-by-sector as it's requested.

If the .ISO image your using public? If you can point me to a download link of an .iso you know does not work, I will give the same disk image a try. Thanks.

**Comment by noagenda33** [comment]
You can try this one. It's only 4MB
[Netboot.xyz.iso](https://www.icloud.com/iclouddrive/0AhtABVf4i1tPSHeI0VK0sQlw#netboot)

**Comment by noagenda33** [comment]
Any update on this? I have applied the few updates that have been released since I posted this bug but SIDER still not working.

**Comment by ke6yjc** [comment]
Was this ever fixed by chance? I'm having some issues trying to re-image a system from remote. OS is Windows 10/11 Pro. Thanks!

**Comment by si458** [comment]
I know the was a fix the other release which was for amt and it included something about iso?
Maybe try the latest version 1.1.30?
I would need access to a system that supports it as my amt 6 and 8 machines don't support it sadly
